### ISSUE TITLE
[Help] I cant find some strings in Weblate

### ISSUE THREAD

**Issue by luckkmaxx** [libre-tube/LibreTube#3927]
### Steps to reproduce

Sorry, I dont know how to PM you instead open this issue.

Yesterday I was trying to edit the strings related to videos upload time, using the search in Weblate. My goal is to set all related strings without the capital letter, because the previous translator set some with it and some without.

I could only find:
- %d year(s) ago
- %d month(s) ago

But I dont know where are the
- %d day(s) ago
- %d week(s) ago
- Yesterday
(I think thats all)

Maybe you know something I dont.
Thanks

### Expected behavior

Not applicable

### Actual behavior

Not applicable

### LibreTube version

Not applicable

### Android version

Not applicable

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I've no clue either, I never really have been translating things on Weblate

**Comment by luckkmaxx** [comment]
Its weird, because these string were in fact translated previously by somebody.
Lets see if somebody else knows something about.

**Comment by luckkmaxx** [comment]
Im pretty sure there  the strings. They gone away and nobody figured out.

**Comment by FireMasterK** [comment]
They're inbuilt into Android iirc. You can't change them. You'll have to submit changes to Android.

**Comment by FireMasterK** [comment]
We use `DateUtils.getRelativeTimeSpanString` if the time is less than a month ago.

**Comment by luckkmaxx** [comment]
> They're inbuilt into Android iirc. You can't change them. You'll have to submit changes to Android.

What?? Do you mean there are strings that are taken directly from android ? Sorry I have no idea what is "iirc"

**Comment by FireMasterK** [comment]
iirc stands for "if I recollect correctly".

https://www.dictionary.com/browse/iirc

**Comment by Bnyro** [comment]
Oh right, I just remember that was a PR by Isira, and the strings are only used on lower Android versions anyways. For days, weeks, and hours, Android always had inbuilt strings, but not for months and years, so that's why they're translatable.
For reference: https://github.com/libre-tube/LibreTube/blob/master/app/src/main/java/com/github/libretube/util/TextUtils.kt#LL92C10-L92C10

**Comment by luckkmaxx** [comment]
Ok.
There is something weird:

The text (in spanish) for less than a month has first capital letter (Hace 2 semanas).
The text for more than a month is all in normal letters (hace 2 meses).

I wanted to fix this, and the weblate strings for months and years **was already** with first capital letter!

**So, after merging in the app, they become all small letters, mismatching with the inbuilt ones.**

I have to say yesterday I changed the "H" to "h" and this was merged, but Im testing with a previous debug apk and stable one.
I will revert this, but will not take effect.
